app/index.js

```diff
@@ -894,8 +915,11 @@ window.onload = function () {
   } else {
     useJoinDialog = false
   }
+  if (queryParams.tokens) {
+    ui.connectDialog.tokens(queryParams.tokens.split(","))
+  }
   if (queryParams.token) {
-    ui.connectDialog.token(queryParams.token)
+    ui.connectDialog.tokens(queryParams.token.split(","))
```

Comment:
Looks like if the same query parameter is given multiple times, the `url` lib (at least the nodejs version) will return an array instead:
```
> url.parse('https://a/?t=1&t=2', true).query.t
[ '1', '2' ]
```
We can probably use that to solve the token-which-contains-comma problem by not splitting `queryParams.token` but instead just adding it raw if it's a string (e.g. `?token=te,st`) or adding each entry raw if it's an array (e.g. `?token=te,st&token=tes,t`).

Reply:
Nice! works perfectly!